<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QPushButton" name="pushButton_getRect">
      <property name="text">
       <string>获取矩形区域</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>571</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3" rowspan="2" colspan="3">
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_createRect">
      <property name="text">
       <string>创建矩形</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>保存xml</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>打开xml</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="saveAct"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionlast"/>
    <addaction name="actionnext"/>
    <addaction name="actiondelete_label"/>
    <addaction name="actionrevise"/>
    <addaction name="actionconfirm"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAct">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionlast">
   <property name="text">
    <string>last</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>next</string>
   </property>
  </action>
  <action name="actiondelete_label">
   <property name="text">
    <string>delete label</string>
   </property>
  </action>
  <action name="actionrevise">
   <property name="text">
    <string>revise</string>
   </property>
  </action>
  <action name="actionconfirm">
   <property name="text">
    <string>confirm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicsRect/graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
